### PR TITLE
Refactor icon path usage

### PR DESCRIPTION
**Pull request by fonurr**

### CODE REVIEW

File: src/common/general/iconPathUtil.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        light: getIconUri(fileName, IconTheme.Light),
+        dark: getIconUri(fileName, IconTheme.Dark)
+    };
+}
+
+export {
+    IconTheme,
+    getIconPath,
+    getIconUri,
+    getThemedIconUris
```
Comment (mertkahyaoglu) [inline src/common/general/iconPathUtil.ts:41]: I think we should export the only function needed here. Exposing others might be confusing. Only exporting `getThemedIconUris` might force users to choose a theme for their icons.

**Review by mertkahyaoglu — APPROVED** [review]

File: src/common/general/iconPathUtil.ts (hunk 1 above)
Comment (fonurr) [reply src/common/general/iconPathUtil.ts:41]: There are places where theme is not used. `getIconPath` is used outside this file. I also exported getIconUri for a possible future use case.

File: src/common/general/iconPathUtil.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import * as vscode from "vscode";
+import * as path from "path";
+import ContextProvider from "../vscode/extension/ContextProvider";
+
+const DEFAULT_LIGHT_RESOURCES_FOLDER_PATH = "resources/light";
+const DEFAULT_DARK_RESOURCES_FOLDER_PATH = "resources/dark";
```
Comment (konukselim) [inline src/common/general/iconPathUtil.ts:6]: Are there any possibility that theme resources could have another folder path? If not so, naming here could be misleading later on.

**Review by konukselim — APPROVED** [review]

File: src/common/general/iconPathUtil.ts (hunk 2 above)
Comment (fonurr) [reply src/common/general/iconPathUtil.ts:6]: currently not, shall I remove the "DEFAULT_" part for now?

Comment (konukselim) [reply src/common/general/iconPathUtil.ts:6]: Hmm, could be better but it's up to you and not that essential 😄 I just wanted to point that.

Comment (fonurr) [reply src/common/general/iconPathUtil.ts:6]: I will remove it 👍

**Review by konukselim — APPROVED** [review]

**Comment by fonurr** [comment]
Force pushed for merging fixup by applying `git rebase -i --autosquash  46e3491`

**Pull request merged**